cupboard_app/v2_urls.py

```diff
@@ -13,7 +13,7 @@
     MeasurementsViewSet,
     UserViewSet,
     UserListIngredientsViewSet,
-    UpdateUserListIngredientsViewSet
+    UpdateUserListIngredientsViewSet,
```

Comment:
Remove the comma as this file is no longer required in this change.

Reply:
done 👍